README.md

```diff
@@ -1,16 +1,201 @@
-# Project
+# Get Quick Assessments Action
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+Get Quick Assessments Action is part of App Compliance Automation Tool(ACAT). With Get Quick Assessments Action you can now get quick assessments of Azure Resources by report or by deployment from your Github Workflows. Since the get quick assessments action can be added directly after the resource deployment, you can have a immediately result of the compliance status of resources deployed this time. Its now even easier to follow safe deployment practices and catch non-compliant issues. 
 
-As the maintainer of this project, please make a few updates:
+New to ACAT? Its an Azure service that help you to get M365 compliance certificate easily. To know more check out: [What is App Compliance Automation Tool(ACAT)](https://learn.microsoft.com/en-us/microsoft-365-app-certification/docs/acat-overview)
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+The definition of this Github Action is in [action.yml]()
 
-## Contributing
+
+# Pre-requisites:
+* Azure Login Action: Authenticate using [Azure Login](https://github.com/Azure/login)  action. The Get Quick Assessments action assumes that Azure Login is done using an Azure service principal that has [sufficient permissions](https://docs.microsoft.com/en-us/azure/governance/policy/overview#rbac-permissions-in-azure-policy) trigger and get quick assessment on the selected scopes. Once login is done, the next set of actions in the workflow can perform tasks such as geting quick assessments by report or by deployment. For more details on permissions, checkout 'Configure credentials for Azure login action' section in this page  or alternatively you can refer the full [documentation](https://github.com/Azure/login) of Azure Login Action.
+* Create an ACAT report(optional): Go to Azure Portal to create an ACAT report for you application.
+* Prepare the deployment id(optional): You can also get quick assessment by your deployment, set the deployment id as output in your former deploy action, and take the deployment id as input of Get Quick Assessments action.
+
+
+
+# Inputs for the Action
+
+* `cred`: mandatory. The credential you use to get quick assessments.
```

Comment:
A question from user's perspective, why do I need this cred when I've already configured the az login action?

Reply:
Should also emphasize cred is same as login action

Reply:
emphasized the cred is the same as login. Do we need extra discussion to decide whether we take cred as input or tenant-id?